packages/runtime/runtime-utils/src/summarizerNode/summarizerNodeWithGc.ts

```diff
@@ -100,6 +103,10 @@ export class SummarizerNodeWithGC extends SummarizerNode implements IRootSummari
             wipSummaryLogger,
         );
 
+        if (config.gcDisabled === true) {
+            this.gcDisabled = true;
+        }
```

Comment:
```suggestion
        this.gcDisabled = config.gcDisabled;
```
Is this not equivalent since they are both `true | undefined` and it is readonly?

Reply:
Yes, it is. The only advantage we get out of the check first is that if its not there, the property is not set at all (even to undefined).

Reply:
Now that I am thinking about this, boolean makes the code clearer and more readable.

Reply:
Changed to boolean.